tests/reporter/kubernetes.go

```diff
@@ -208,7 +208,7 @@ func (r *KubernetesReporter) logDomainXMLs(virtCli kubecli.KubevirtClient, vmis
 		if vmi.IsFinal() {
 			continue
 		}
-		domxml, err := libvmi.GetRunningVirtualMachineInstanceDomainXML(virtCli, &vmi, r.namespace)
+		domxml, err := libvmi.GetRunningVirtualMachineInstanceDomainXML(virtCli, &vmi, r.namespaces[0])
```

Comment:
This looks weird. 

I wonder: you have the vmi. Can't you get (and use) the VMI's namespace?